generators/languages/USAGE

```diff
@@ -0,0 +1,24 @@
+Description:
+    Select languages from a list of available languages. The i18n files will be copied to the /webapp/i18n folder.
+
+Example:
+    jhipster languages
+
+    This will create for each selected language:
+        /webapp/i18n/{lang}/global.json
+        /webapp/i18n/{lang}/activate.json
+        /webapp/i18n/{lang}/audits.json
+        /webapp/i18n/{lang}/configuration.json
+        /webapp/i18n/{lang}/error.json
+        /webapp/i18n/{lang}/gateway.json
+        /webapp/i18n/{lang}/health.json
+        /webapp/i18n/{lang}/home.json
```

Comment:
We have to override home.json file because there is NHipster customization messages for client, for example: https://github.com/jhipster/generator-jhipster-nodejs/blob/20d3b7d748dbd0db565833edb422f03163cd926b/generators/client/templates/angular/home/home.component.html.ejs#L20

Reply:
Ok, I see. I will change all above files. Thank you.